source/tutorials/metadata.rst

```diff
@@ -0,0 +1,138 @@
+Metadata in QIIME 2
+===================
+
+Metadata enables users of QIIME 2 to supplement their analyses with information relevant to their study. This manifests itself in many ways: indicating the barcode sequence when demultiplexing sequences, sorting and grouping taxonomic assignments in the taxonomic bar plots, or exploring diversity metrics with visualizations like ``alpha-group-significance`` or ``beta-group-significance``. In QIIME 1, metadata (also known as the metadata mapping file), was a user-defined file that contained these study-specific fields. QIIME 2 expands upon this idea, allowing users to provide their own study metadata (via a TSV file), or, by viewing other study-specific artifacts as metadata. Examples of both are presented in the following.
+
+Begin by creating a directory to conduct the work in:
+
+.. command-block::
+   :no-exec:
+
+   mkdir qiime2-metadata-tutorial
+   cd qiime2-metadata-tutorial
+
+Metadata from a mapping file
```

Comment:
What do you think of avoiding using the term "mapping file" in favor of "[sample|feature] metadata mapping file", since "mapping file" is so generic. I've been trying to do this in my slides, forum posts, etc.